src/Entities/RegistryContext.h

```diff
@@ -0,0 +1,55 @@
+#include <Graphics/DebugLines.h>
```

Comment:
Don't know how much @handsomematt cares, but I always add the copyright header to new files